test/AutoRest.TestServer.Tests/body-array.cs

```diff
@@ -100,9 +98,11 @@ public class BodyArray : TestServerTestBase
         });
 
         [Test]
-        public Task GetArrayByteWithNull() => Test((host, pipeline) =>
+        public Task GetArrayByteWithNull() => Test(async (host, pipeline) =>
         {
-            Assert.ThrowsAsync(Is.InstanceOf<Exception>(), async () => await new ArrayClient(ClientDiagnostics, pipeline, host).GetByteInvalidNullAsync());
+            var result = await new ArrayClient(ClientDiagnostics, pipeline, host).GetByteInvalidNullAsync();
```

Comment:
Test says invalid but we don't throw.

Reply:
It is valid for us since `byte[]` is able to be null. Don't trust the test server names as gospel.